.gitignore

```diff
@@ -3,6 +3,7 @@ _site
 
 # Ignore ruby files
 .bundle
+.tool-versions
```

Comment:
I would not commit these kind of files. For me it's like committing the .vscode folder.

You can use the gitignore-global for your dependency management app.

Reply:
Updated, thanks!